contrib/sync_namespaces_jobs.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+import os
+
+import yaml
+
+DST_DIR = "/tmp/tron-servdir"
+#DST_DIR = "/nail/home/chl/tron-servdir"
+SRC_DIR = "/nail/etc/services/tron/prod"
+
+#SRC_DIR = "/nail/home/chl/git/yelpsoa-configs/tron/prod"
+#DST_DIR = "/nail/home/chl/Tron/contrib/tron-servdir"
+
+
+def main():
+    print(
+        "This script will load the same amount of namepance and jobs from tron-prod to tron-playground."
+    )
+    print("Please make sure you run it at playground")
+
+    for (dirpath, dirname, filenames) in os.walk(SRC_DIR):
```

Comment:
nit: listdir, since you're assuming that every file is directly in SRC_DIR (e.g. located at SRC_DIR/filename) anyway. Then you can get rid of this outer loop.